apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -177,7 +180,10 @@ const MeetingForm: React.FC = () => {
         label="Enable Web Audio"
         value=""
         checked={isWebAudioEnabled}
-        onChange={toggleWebAudio}
+        onChange={(): void => {
+          toggleWebAudio();
+          setMeetingMode(MeetingMode.Attendee);
```

Comment:
Why doesn't WebAudio not work with spectator mode?

Reply:
Basically some weird error because we havent called getUserMedia at all, the attendee is somehow dropped from the meeting. So we thought it doesnt make sense to have both of these checkboxes enabled at the same time anyways so just disable one when the other is checked.

Reply:
Can we reproduce the issue in JS SDK? Feel like we need to know why the attendee getting dropped.

Reply:
Hard to say for sure, I haven't found a good way to reproduce the issue the SDK. I was going to create a sim to track separately to root cause better and depending on the root cause generate a warning.

Reply:
So is there a need to fix this right now before we root cause it?

Reply:
Technically this isnt release-blocking right? If so, then maybe its not needed right now actually

Reply:
We could leave it for now, but it's not very good behavior and would be good to at least put in a temporary fix. If the user joins the meeting with WebAudio which is on by default and Disable Audio Video, they don't see the roster and can't hear the meeting.

Reply:
it is not release blocking so I would vote for root cause it first :)

Reply:
Ok, I'll revert this fix and create a sim to track

Reply:
Updated, now the fix just enables webaudio properly